source/openshift_route.go

```diff
@@ -259,18 +262,35 @@ func (ors *ocpRouteSource) endpointsFromOcpRoute(ocpRoute *routev1.Route, ignore
 	return endpoints
 }
 
-func (ors *ocpRouteSource) targetsFromOcpRouteStatus(status routev1.RouteStatus) endpoint.Targets {
-	var targets endpoint.Targets
+// targetsAndHostFromOcpRouteStatus returns the router's canonical hostname and host
+// either for the given router if it admitted the route
+// or for the first (in the status list) router that admitted the route.
+func (ors *ocpRouteSource) targetsAndHostFromOcpRouteStatus(status routev1.RouteStatus) (endpoint.Targets, string) {
```

Comment:
`getTargets` personally would be a more apt name. We don't need to describe how in the function name.

Reply:
Changed to `getTargetsFromRouteStatus` so that it looks consistent with [getTargetsFromTargetAnnotation](https://github.com/kubernetes-sigs/external-dns/blob/master/source/source.go#L206).